bin/tictactoe

```diff
@@ -1,3 +1,34 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+
+def call 
+  puts "How many players? (1, 2, or 0 = [yourself])"
+  input = gets.strip
+  
+  if input == "1"
+  
+  elsif input == "2"
+  
+  elsif input == "0"
+  
+  else
+  
+  end
+end
+
+def again?
+  puts "Want to play again?(Yn)"
+  input = gets.strip.upcase
+  if input == "Y"
+    call
+  elsif input == "N"
+    exit
+  else
+    puts "A Valid Choice !!"
+    again?
+  end
+end
+
+call
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.